lib/storage/http.js

```diff
@@ -16,6 +16,8 @@ const http = require('@adobe/httptransfer');
 const { AssetComputeLogUtils, GenericError, RenditionTooLarge } = require('@adobe/asset-compute-commons');
 const { actionName } = require('../action');
 
+const MAX_RETRY_DURATION_UPLOAD = 900000; // 15 mins
```

Comment:
note: before this change, this value was the default 60s

yes we could get fancy and determine this value based on how much time is left in the action, etc, but I went with the simplest solution that fixes the targetted bug

Reply:
ACK